build.open-xml-sdk.cmd

```diff
@@ -0,0 +1,18 @@
+@echo off
+
+REM Restore NuGet packages
+.nuget\NuGet.exe restore Open-XML-SDK.sln
```

Comment:
This will restore all NuGet packages, including the console test runner used below to run the tests.